README.md

```diff
@@ -3,75 +3,20 @@
 [![crates.io version][1]][2] [![build status][3]][4]
 [![downloads][5]][6] [![docs.rs docs][7]][8]
 
-> Rust version of [flat-tree](https://github.com/mafintosh/flat-tree). A series of functions to map a binary tree to a list.
+Rust version of [mafintosh/flat-tree](https://github.com/mafintosh/flat-tree).
+Map a binary tree to a list.
 
-## Usage
+- [Documentation][8]
+- [Crates.io][2]
 
-``` rs
+## Usage
+```rust
 extern crate flat_tree;
 
 let parent = flat_tree::parent(0);
 println!("parent of 0 is {}", parent);
 ```
 
-## API
-
```

Comment:
I'm aware of that this is duplicate information but still a bit sad to see it go.